odoo/tests/common.py

```diff
@@ -85,6 +85,8 @@
     websocket = None
 
 _logger = logging.getLogger(__name__)
+_logger.info("Importing test framework", stack_info=_logger.isEnabledFor(logging.DEBUG))
```

Comment:
+ 1 for the general idea, but could you add a raise or a log error in this file `if not config['test_enable']` to avoid an accidental import from business code??

Reply:
@Xavier-Do  Something like?
```suggestion
_logger.log(logging.INFO if config['test_enable'] else logging.ERROR, "Importing test framework", stack_info=_logger.isEnabledFor(logging.DEBUG) or not config['test_enable'])
```

Reply:
Yes why not, but we could also specify why this is an error. But looks good to me ;)

Reply:
Done